.github/workflows/itests.yml

```diff
@@ -37,11 +37,11 @@ jobs:
       GOOS: linux
       GOARCH: amd64
       GOPROXY: https://proxy.golang.org
-      DAPR_CLI_VER: 1.9.1
-      DAPR_RUNTIME_VER: 1.10.5
-      DAPR_INSTALL_URL: https://raw.githubusercontent.com/dapr/cli/3dacfb672d55f1436c249057aaebbe597e1066f3/install/install.sh
+      DAPR_CLI_VER: 1.12.0
+      DAPR_RUNTIME_VER: 1.12.0
+      DAPR_INSTALL_URL: https://raw.githubusercontent.com/dapr/cli/release-1.12/install/install.sh
       DAPR_CLI_REF: ''
-      DAPR_REF: '4181de0edc65fc98a836ae7abc6042c575c8fae5'
+      DAPR_REF: '2149fca96cdf11627c387bda26dcc027d1c47354'
```

Comment:
The Dapr ref set's a specific commit and is redundant to the 1.12 flag. I'd prefer we just set 1.12 and leave that blank.